helm/bankapp/templates/mysqlStatefulSet.yml

```
@@ -0,0 +1,67 @@
+            - -h
+            - localhost
+          initialDelaySeconds: 10
+          periodSeconds: 5
+  volumeClaimTemplates:
+  - metadata:
+      name: "{{ .Values.db_statefulset.name }}-data"
+      labels:
+        app: {{ .Values.db_statefulset.name }}
+    spec:
+      accessModes:
+        - ReadWriteOnce
```

yamllint (1.35.1)

[warning] wrong indentation: expected 6 but found 8

(indentation)